### PR TITLE
Update to 2.12.3

### PR DESCRIPTION
**Pull request by markolemmetty**
Update to fix CVE-2023-45322.

### CODE REVIEW

**Review by vigejolla — APPROVED** [review]
It's a bit hard to follow what was changed in the patch, as github does not seem to realize it's the same file. I suppose it's ok though.

**Comment by markolemmetty** [comment]
Ok, I did change the patch file name.

**Comment by vigejolla** [comment]
> Ok, I did change the patch file name.

Thanks, indeed now github shows the diff much better. Still it looks good to me - as long as it builds.

File: rpm/0001-Disable-documentation-further.patch [hunk 1]
```diff
@@ -1,41 +1,37 @@
 From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
-From: Pekka Vuorela <pekka.vuorela@jolla.com>
-Date: Tue, 10 Jan 2023 18:00:06 +0200
-Subject: [PATCH] Disable documentation further.
+From: Marko Lemmetty <marko.lemmetty@jolla.com>
+Date: Fri, 22 Dec 2023 14:53:38 +0200
+Subject: [PATCH] [PATCH] Disable documentation further
```
Comment (mlehtima) [inline rpm/0001-Disable-documentation-further.patch]: this seems to have duplicate PATCH tag

Comment (markolemmetty) [reply rpm/0001-Disable-documentation-further.patch]: Yes, so it seems. I will remove it.

**Review by mlehtima — APPROVED** [review]

**Pull request merged**